docs/user_needs/zephyr_rtos_user.sdoc

```diff
@@ -0,0 +1,87 @@
+[DOCUMENT]
+TITLE: Zephyr RTOS User
+REQ_PREFIX: ZEP-
+
+[GRAMMAR]
+IMPORT_FROM_FILE: user_needs.sgra
+
+[FREETEXT]
+
+TBD
+
+[/FREETEXT]
+
+[USER_NEED]
+UID: ZEP-127
+STATUS: Draft
+TYPE: User Need
+USER: Product Manufacturer
+TITLE: OSS License Allows Statically Linked Private Extensions
+STATEMENT: >>>
+As a Product Manufacturer of resource-constrained devices,
+
+I need open-source software to be licensed such that the proprietary code statically linked with the open source software can be kept private.
+<<<
+
+[USER_NEED]
+STATUS: Draft
+TYPE: User Need
+USER: Zephyr RTOS User
+TITLE: Easily Switch Application to different MCU Architecture
+STATEMENT: >>>
+As a Zephyr RTOS user,
+
+I want to be able to easily switch my application to a different MCU architecture (x86, ARM Cortex-M/A, RISCV etc.).
+<<<
+
+[USER_NEED]
+UID: ZEP-128
+STATUS: Draft
+TYPE: User Need
+USER: Product Manufacturer
+TITLE: Development Systems Options for Windows, Linux
+STATEMENT: >>>
+As a Product Manufacturer,
+
+I need embedded software development system options for Windows and Linux host operating systems.
```

Comment:
I like the statement, just want to mention that many embedded developers use Macs. I am 96% sure that Zephyr would not want to exclude them.

Reply:
Also, in my understanding, supporting all three of Windows, Linux and Mac is a minimum requirement for claiming portability across OSes when it comes to development.

Reply:
I expected this would generate some discussion. The key question is whether to list each of the *three* as a separate need or combine them into one.

Reply:
INCOSE rules advise to not include incomplete lists. Maybe the list of architectures could go to a COMMENT.

At least the three major versions should be supported: Windows, popular Linux (e.g., Ubuntu), macOS (latest release).